terraform/cloud-platform/bastion.tf

```diff
@@ -2,7 +2,7 @@ locals {
   bastion_fqdn = "bastion.${local.cluster_base_domain_name}"
 
   # The URL below is generated by the authorized-keys-provider app running on cloud-platform-live-0.
```

Comment:
Please change the cluster name in the comment, now that it's been moved to live-1